experiment/molecule/src/common.ts

```diff
@@ -14,53 +15,268 @@ export const Uint8: FixedBinaryCodec<number> = {
     return new DataView(buf).getUint8(0);
   },
 };
-export declare const HexUint8: FixedBinaryCodec<string>;
+export const HexUint8: FixedBinaryCodec<string> = {
+  __isFixedCodec__: true,
+  byteLength: 1,
+  pack(str) {
+    return Uint8.pack(parseInt(str, 16));
+  },
+  unpack(buf) {
+    return `0x${Uint8.unpack(buf).toString(16)}`;
+  },
+};
 
 // array Uint16 [byte; 2]
 /**
  *
  * @alias Uint16LE
  */
-export declare const Uint16: FixedBinaryCodec<number>;
-export declare const Uint16LE: FixedBinaryCodec<number>;
-export declare const Uint16BE: FixedBinaryCodec<number>;
+export const Uint16LE: FixedBinaryCodec<number> = {
+  __isFixedCodec__: true,
+  byteLength: 2,
+  pack(u8) {
+    const buf = new ArrayBuffer(2);
+    new DataView(buf).setUint16(0, u8, true);
+    return buf;
+  },
+  unpack(buf) {
+    return new DataView(buf).getUint16(0, true);
+  },
+};
+export const Uint16 = Uint16LE;
+export const Uint16BE: FixedBinaryCodec<number> = {
+  __isFixedCodec__: true,
+  byteLength: 2,
+  pack(u8) {
+    const buf = new ArrayBuffer(2);
+    new DataView(buf).setUint16(0, u8);
+    return buf;
+  },
+  unpack(buf) {
+    return new DataView(buf).getUint16(0);
+  },
+};
 /**
  * @alias HexUint16LE
  */
-export declare const HexUint16: FixedBinaryCodec<string>;
-export declare const HexUint16LE: FixedBinaryCodec<string>;
-export declare const HexUint16BE: FixedBinaryCodec<string>;
+export const HexUint16LE: FixedBinaryCodec<string> = {
```

Comment:
wrong alias JSDoc here, would please add a correct `@alias` for each alias Codec?

Reply:
done